src/geometry/GeometryUtils.h

```diff
@@ -8,6 +8,7 @@ using Polygons = std::vector<Polygon>;
 
 using IndexedFace = std::vector<int>;
 using IndexedTriangle = Vector3i;
+using PolygonsInd = std::vector<IndexedFace>;
```

Comment:
IMO better to spell out non-standard abbreviations -> PolygonIndices.
Otoh, since this is used only once, perhaps drop the alias and just use `std::vector<IndexedFace>` directly in PolySet?